sdk/python/kfp/_client.py

```diff
@@ -213,22 +202,22 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
     Returns:
       A run object. Most important field is id.
     """
-    import kfp_run
+    import kfp_server_api
```

Comment:
imports should all happen at the top of the package.

Reply:
Yes. I'll move it to the top.

Reply:
Done.